packages/flutter/test/material/navigation_rail_test.dart

```diff
@@ -0,0 +1,1409 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Should the new files be 2020?

Reply:
I think they are all now suppose to be 2014 (the analyzer was complaining)